crates/bevy_ui/src/render/box_shadow.rs

```diff
@@ -479,7 +479,26 @@ pub fn prepare_shadows(
                         box_shadow.radius.bottom_left,
                     ];
 
-                    let uvs = [Vec2::ZERO, Vec2::X, Vec2::ONE, Vec2::Y];
+                    let uvs = [
+                        Vec2::new(
+                            uinode_rect.min.x + positions_diff[0].x,
+                            uinode_rect.min.y + positions_diff[0].y,
+                        ),
+                        Vec2::new(
+                            uinode_rect.max.x + positions_diff[1].x,
+                            uinode_rect.min.y + positions_diff[1].y,
+                        ),
+                        Vec2::new(
+                            uinode_rect.max.x + positions_diff[2].x,
+                            uinode_rect.max.y + positions_diff[2].y,
+                        ),
+                        Vec2::new(
+                            uinode_rect.min.x + positions_diff[3].x,
+                            uinode_rect.max.y + positions_diff[3].y,
+                        ),
+                    ]
+                    .map(|pos| pos / uinode_rect.max);
```

Comment:
Should this be `max`? or should it be `size()`? I think a non-zero x_offset or y_offset will show an error.